.circleci/config.yml

```diff
@@ -1,32 +1,34 @@
-deps-run: &deps-install
-  name: Install Python dependencies and PlasmaPy
-  command: |
-    pip install --user -r requirements/automated-documentation-tests.txt | cat
-    pip install --user . | cat
-
-html-run: &doc-html
-  name: Build HTML documentation
-  command: python setup.py build_docs -w
-
-latex-run: &doc-latex
-  name: Build LaTeX documentation
-  command: python setup.py build_docs -b latex -w
-
 version: 2
 jobs:
   test-html:
     docker:
       - image: plasmapy/documentation-builder:latest
     steps:
       - checkout
-      - run: *deps-install
-      - run: *doc-html
+      - run: 
+          name: Install Python dependencies and PlasmaPy
+          command: |
+            pip install --user -r requirements/automated-documentation-tests.txt | cat
+            pip install --user . | cat
+      - run:
+          name: Build HTML documentation
+          shell: /bin/bash
+          command: |
+            python setup.py build_docs -W 2> >(tee /tmp/stderr) 1> >(tee /tmp/stdout) || true; echo $? | tee /tmp/EXITCODE
```

Comment:
Nice!

Reply:
Thanks, I'm rather proud of this little random-walk-spawned monstrosity ^^